auth0pull/auth0pull.go

```diff
@@ -12,9 +12,9 @@ import (
 
 	"cloud.google.com/go/datastore"
 	stackdriver "cloud.google.com/go/logging"
+	"github.com/ajvb/auth0/management"
```

Comment:
Do we have any plans to PR a fix in for this, or wait to hear back on the issue itself I guess?

(just wondering how we will track upstream or if it's required)

Reply:
@ameihm0912 Yeah, just waiting to hear back on the issue. Once I hear back I will either PR a fix or hopefully be able to just switch back to the main `go-auth0` client lib.